app/components/pages/Dashboard/index.js

```diff
@@ -29,6 +29,7 @@ const MobileOnlyStickyFooter = styled(StickyFooter)`
 `
 
 const CenterdSmallParagraph = styled(SmallParagraph)`
+  margin-top: ${th('space.7')};
```

Comment:
This spacing (space.7) is much larger than it used to be (space.2). It has this affect on the submissions tab:
![image](https://user-images.githubusercontent.com/15656538/47804612-29d6a600-dd2d-11e8-8b03-d574c1669e9c.png)

Was this intentional?

Also, please see the last PR comment about `margin-top`. This issue is probably best resolved by applying a `min-height` to the `TabbedSubmissions` component & leaving the existing `margin-bottom` (unless it needs to be increased to `space.7`?)

Reply:
@jsms90 sorry, my bad, I will fix it now

Reply:
no problem at all :+1: